janusgraph-test/src/test/java/org/janusgraph/graphdb/JanusGraphFactoryShorthandTest.java

```diff
@@ -16,6 +16,8 @@
 
 import org.janusgraph.core.JanusGraphFactory;
 import org.janusgraph.core.JanusGraph;
+import org.janusgraph.graphdb.management.JanusGraphManager;
+import org.apache.tinkerpop.gremlin.server.Settings;
```

Comment:
Unused imports (JanusGraphManager and Settings) in this and other test classes below now that initialization was removed.